src/index.js

```diff
@@ -76,8 +76,12 @@ export default class AlphaScrollFlatList extends Component {
 
                 index = this.props.reverse ? lastIndex : firstIndex;
             } else {
-                //Get index of item with that letter and scroll to the first result on the list
-                index = this.props.data.findIndex(item => item[this.props.scrollKey].charAt(0).localeCompare(letter) === 0);
+                if(typeof this.props.scrollKey === 'function') {
+                    index = this.props.data.findIndex(item => this.props.scrollKey(item).charAt(0).localeCompare(letter) === 0);
```

Comment:
We should avoid repeating code, this could be as simple as:
```
const scrollKeyIsFunction = typeof this.props.scrollKey === 'function';
index = this.props.data.findIndex(item =>(scrollKeyIsFunction ? this.props.scrollKey(item) : item[this.props.scrollKey]).charAt(0).localeCompare(letter) === 0);
```